golang/cosmos/x/swingset/genesis.go

```diff
@@ -34,13 +42,25 @@ func InitGenesis(ctx sdk.Context, keeper Keeper, data *types.GenesisState) []abc
 	// Just run the SwingSet kernel to finish bootstrap and get ready to open for
 	// business.
 	stdlog.Println("Running SwingSet until bootstrap is ready")
-	bootstrapBlock := abci.RequestEndBlock{Height: ctx.BlockHeight()}
-	valUpdate, err := EndBlock(ctx, bootstrapBlock, keeper)
```

Comment:
Just one thing to double-check. This call sets `endBlockHeight` and `endBlockTime` in `abci.go`. Assuming that `ctx.BlockHeight()` is zero, the behavior is unchanged, but the `endBlockTime` now remains uninitialized. I *think* that's fine, as there should be a (real) call to `EndBlock()` before the next call to `CommitBlock()`, which is the only thing that reads those fields, but please double-check to be sure.

Reply:
That understanding is correct.